arch/x86/apic.c

```diff
@@ -181,7 +180,7 @@ void init_apic_timer(void) {
         apic_write(APIC_LVT_TIMER, timer.reg);
 
         /* Sleep for 100ms to calibrate, count the ticks */
-        pit_sleep(100);
+        sleep(100);
```

Comment:
Side note: let's maybe change the value to 10ms. Because 100ms x 10 takes a while.